openshift/backup/backup-deploy.yaml

```diff
@@ -139,10 +139,10 @@ objects:
                 claimName: ${VERIFICATION_VOLUME_NAME}
             - name: ${NAME}-config-volume
               configMap:
-                name: ${CONFIG_MAP_NAME}
+                name: backup-config
                 items:
-                  - key: ${CONFIG_FILE_NAME}
-                    path: ${CONFIG_FILE_NAME}
+                  - key: backup.conf
+                    path: backup.conf
```

Comment:
aren't these changes just the same as what is mentioned in the params down below? 🤔

Reply:
not sure what was wrong with the template but the command wasn't recognizing what was being piped in OR the default values.

Again, I'm just capturing the changes I made to get the container running.